notebook/tests/selenium/test_find_and_replace_apply_all.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
Let's call the file `test_find_and_replace.py` - we'll probably later want to add tests that aren't doing apply all.